code/game/gamemodes/objective.dm

```diff
@@ -35,6 +35,9 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 /datum/objective/proc/admin_edit(mob/admin)
 	return
 
+/datum/objective/proc/on_obj_given()
+	return
```

Comment:
what about when someone is removed from the objective?

Reply:
no longer used as that way